grunttasks/l10n-extract.js

```diff
@@ -22,10 +22,13 @@ module.exports = function (grunt) {
 
     var clientWalker = extract({
       'input-dir': path.join(__dirname, '..', '.es5', 'scripts'),
-      'join-existing': false,
+      joinExisting: true,
       'keyword': ['t', 'unsafeTranslate'],
       'output': 'client.pot',
+      // output-dir is used by jsxgettext-recursive
```

Comment:
Looks like @zaach merged your PR for `outputDir` - https://github.com/zaach/jsxgettext-recursive/pull/7, thanks @zaach!